src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
```

Comment:
The same issue with the `logo` variable. You are using it only in the `promise1` promise. So it's better to move this variable into this promise.